src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -332,10 +332,10 @@ function MdProgressCircularDirective($window, $mdProgressCircular, $mdTheming,
    * @param {number} value Percentage of circle (between 0 and 100)
    * @param {number} limit Max percentage for circle
    *
-   * @returns {number} Stroke length for progres circle
+   * @returns {number} Stroke length for progress circle
    */
   function getDashLength(diameter, strokeWidth, value, limit) {
-    return (diameter - strokeWidth) * $window.Math.PI * ((3 * (limit || 100) / 100) - (value/100));
+    return getCircumference(diameter, strokeWidth,((-value/100) - (limit || 100) / 100));
```

Comment:
```suggestion
    return getCircumference(diameter, strokeWidth, ((-value/100) - (limit || 100) / 100));
```
What's `-value` doing here? Can do we anything to make this calculation more clear?

Reply:
I simplified the calculation. Hopefully it is clearer now.